x/blob/types/payforblob.go

```diff
@@ -21,6 +22,20 @@ import (
 const (
 	URLMsgPayForBlobs = "/celestia.blob.v1.MsgPayForBlobs"
 	ShareSize         = appconsts.ShareSize
+
+	// PFBGasFixedCost is a rough estimation of the gas that arises from several other places:
+	// - signature verification
+	// - tx size
+	// - read access to accounts
+	// From modelling PFB gas consumption we arrive at the following formula:
+	// 8 * 512 * shares + 65000. This has a correlation coefficient of 0.996
```

Comment:
[question] In this context does `shares` refer to the # of shares occupied by the blob? If yes, I interpret `8 * 512 * shares` as a variable cost so it's not clear to me why this impacts `PFBGasFixedCost`

Reply:
This is the linear model y = mx + c. The c component is extracted out into `PFBGasFixedCost`. Does that make sense?